src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,211 @@
+                            connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            return statement.executeUpdate() != 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t delete car by id " + id + "from DB ", e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        List<Car> cars = new ArrayList<>();
+        String query = "SELECT c.id car_id, c.model, m.id model_id,"
+                + " m.country , m.name"
+                + " FROM cars_drivers LEFT JOIN cars c ON c.id = cars_drivers.cars_id"
+                + " LEFT JOIN manufacturers m on c.manufacturers_id = m.id"
+                + " WHERE drivers_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                    PreparedStatement statement =
+                            connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(getCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all cars from DB by driver id "
+                    + driverId, e);
+        }
+        return cars;
+    }
+
+    private void insertDrivers(Car car) {
+        String query = "INSERT INTO `cars_drivers` (cars_id, drivers_id) VALUE (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                    PreparedStatement statement =
+                            connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                statement.setLong(2, driver.getId());
+                statement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t insert all drivers to car: " + car, e);
+        }
+    }
+
+    private List<Driver> getDriverForCar(Long id) {
+        List<Driver> driverList = new ArrayList<>();
+        String query = "SELECT d.id id, d.name, d.license_number license "
+                + "FROM cars_drivers cd LEFT JOIN drivers d on d.id = cd.drivers_id"
+                + " WHERE cd.cars_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                    PreparedStatement statement =
+                            connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                driverList.add(getDriver(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all drivers from DB by car id " + id, e);
+        }
+        return driverList;
+    }
+
+    private void deleteDrivers(Long id) {
```

Comment:
it's not necessary to change the name of this method but it doesn't delete any drivers